Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.py

```diff
@@ -103,11 +103,11 @@ def ssh_execute(command: str):
 def scp_execute(file_name: str, file_path: str):
     if SCP_EXTRA_PARAMS:
         param_list = ['scp', '-o', 'StrictHostKeyChecking=no', '-i', CERTIFICATE_FILE.name] + SCP_EXTRA_PARAMS + [
-            file_name, USERNAME + '@' + HOSTNAME + ':' + file_path]
+            file_name, USERNAME + '@' + HOSTNAME + ':' + f'\'{file_path}\'']
```

Comment:
Why would you want to add `'` before and after the file path instead of just using it?

Reply:
Caller sets file_name and file_path like below for scp_execute().
file_name: **Remediation - URL EDL.txt.txt**
file_path: **/var/www/html/files/Demisto Remediation - URL EDL.txt** (including destination filename)

Then, the command line is like below:
**scp Demisto Remediation - URL EDL.txt.txt user@host:/var/www/html/files/Demisto Remediation - URL EDL.txt**

In that case, scp understands that distination is **/var/www/html/files/Demisto**
Actually, !pan-os-edl-update doesn't work without quote.
![image](https://user-images.githubusercontent.com/54964121/67348762-efd76300-f580-11e9-841e-f823499b75fc.png)

Right command line should be as below:
**scp 'Demisto Remediation - URL EDL.txt.txt' user@host:'/var/www/html/files/Demisto Remediation - URL EDL.txt'**

So, file_path need to be quoted.